auslib/admin/views/permissions.py

```diff
@@ -26,6 +26,21 @@ def get(self):
         return jsonify(dict(users=users))
 
 
+class CurrentUserView(AdminView):
+    """Returns all of the details about the logged in user. The UI needs this
+    method to know things about the current user because it does not have
+    access to REMOTE_USER, so it cannot query directly by name."""
+
+    def get(self, username):
+        if username == "current":
+            username = request.environ.get('REMOTE_USER', request.environ.get("HTTP_REMOTE_USER"))
+        permissions = dbo.permissions.getUserPermissions(username)
```

Comment:
This would be to try to help minimize the possibility of privilege escalations? Eg: someone gets in as ffxbld, and then uses this endpoint to figure out which account to target to gain admin access?